widgets/contact/contact.php

```diff
@@ -948,15 +949,20 @@ function render_form_fields( $fields, $errors = array(), $instance ) {
 		}
 	}
 
-	function render_form_label( $field_id, $label, $position ) {
+	function render_form_label( $field_id, $label, $position, $required ) {
 		if ( ! empty( $label ) ) {
 			$label_class = '';
 			if ( ! empty( $position ) ) {
 				$label_class = ' class="sow-form-field-label-' . $position . '"';
 			}
-			?><label<?php if ( ! empty( $label_class ) ) {
-				echo $label_class;
-			} ?> for="<?php echo esc_attr( $field_id ) ?>"><strong><?php echo esc_html( $label ) ?><span class="sow-form-required">*</span></strong></label>
+
+			if ( $required ) {
+				$required = '<span class="sow-form-required">*</span>';
```

Comment:
Don't hijack the `$required` variable. Rather declare a new one for the output, maybe `$required_indicator`.